MoyaLogger.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+
+
+public struct MoyaLogger: Logger {
+    
+    private let loggerId = "Moya_Logger"
+    private let dateFormatString = "dd/MM/yyyy HH:mm:ss"
+    private let dateFormatter = NSDateFormatter()
+    
+    private var date: String {
+        
+        dateFormatter.dateFormat = dateFormatString
+        dateFormatter.locale = NSLocale(localeIdentifier: "en_US_POSIX")
+        return dateFormatter.stringFromDate(NSDate())
+    }
+    
+    public func logNetworkRequest(request: NSURLRequest) {
+        
+        let requestOutput = String(format: "%@: [%@] Request:  %@", loggerId, date, request.description)
+        print(requestOutput)
+        
+        if let headers = request.allHTTPHeaderFields {
+            let requestHeadersOutput = String(format: "%@ [%@] Request Headers:  %@", loggerId, date, headers)
+            print(requestHeadersOutput)
+        }
+        
+        if let bodyStream = request.HTTPBodyStream {
```

Comment:
Yup! I don't think we need to go overboard – people are free to write their own – I'll leave it up to you.